_assets/javascripts/components/jumbotron-video.js

```diff
@@ -1,5 +1,24 @@
 (function() {
   window.$(document).ready(function() {
     new window.CRDS.JumbotronVideos();
+
+    // Prevent background video from loading on mobile
+    const mediaQuery = window.matchMedia('(min-width: 480px)');
+    mediaQuery.addEventListener('change', handleScreenChange);
+
+    // Check screen size on load
+    handleScreenChange(mediaQuery);
+
+    function handleScreenChange(e) {
+      var bgVideo = document.getElementById('jumbotron-bg-video');
+
+      if (e.matches) {
+        bgVideo.setAttribute('src', bgVideo.getAttribute('data-src'));
+      } else {
+        if (!bgVideo.hasAttribute('data-src')) {
+          bgVideo.setAttribute('data-src', bgVideo.getAttribute('src'));
+        }
+      }
```

Comment:
Getting errors on pages without the jumbotron video
![Screen Shot 2021-06-30 at 2 07 16 PM](https://user-images.githubusercontent.com/32345656/124010160-7bd39c00-d9ac-11eb-9e32-df7f453d66fc.png)

Reply:
@silasbaur Fixed!